packages/viewer/src/components/scene-tree-table-layout/scene-tree-table-layout.tsx

```diff
@@ -357,7 +357,11 @@ export class SceneTreeTableLayout {
   private async computeAndUpdateViewportRows(): Promise<void> {
     this.computeViewportRows();
 
-    if (this.controller?.isConnected) {
+    if (
+      this.controller?.isConnected &&
+      this.viewportStartIndex >= 0 &&
```

Comment:
Yeah, this is the case specifically when the `this.totalRows` value is `0`, the `viewportEndIndex` gets set to `-1` and causes some problems in the controller. This could likely be changed to `this.totalRows > 0` and have the same effect (and makes more sense), so I'll update this